assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -236,6 +244,11 @@ function MediaPane(props) {
   const resources = media.filter(filterResource);
 
   const onSearch = (value) => {
+    if (value.trim()) {
+      setIsSearching(true);
+    } else {
+      setIsSearching(false);
+    }
     setSearchTerm({ searchTerm: value });
```

Comment:
Might be a good idea to do `value.trim()` here. No point in searching for empty strings.

Reply:
Done